components/03-modules/header/_featured-info.scss

```diff
@@ -0,0 +1,49 @@
+$featured-info__title-fontsize   : 12px !default;
+$featured-info__subtitle-fontsize: 10px !default;
+
+.featured-info {
+    display: flex;
+    margin: 8px 16px;
+    color: $font-color-base;
+    text-decoration: none;
+    &:focus,
+    &:hover {
+        text-decoration: none;
+    }
+
+    &__icon {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-right: 8px;
+
+        &-svg {
+            width: 24px;
+            height: 24px;
+            fill: $color-primary;
```

Comment:
move color-primary to internal variable and then use it here

Reply:
done